docs/docs/reference/settings.mdx

```diff
@@ -706,6 +706,26 @@ export MELTANO_STATE_BACKEND_LOCK_RETRY_SECONDS=720
 
 ---
 
+### <a name="state-backend-uri"></a>`state_backend.azure.storage_account_url`
+
+- [Environment variable](/guide/configuration#configuring-settings): `MELTANO_STATE_BACKEND_AZURE_STORAGE_ACCOUNT_URL`
+- Default: None
+
+One can [Sign-in to Azure](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-python?tabs=managed-identity%2Croles-azure-portal%2Csign-in-azure-cli#sign-in-and-connect-your-app-code-to-azure-using-defaultazurecredential) using the Azure CLI.
+
+The `storage_account_url` which looks like this `https://<myStorageAccountName>.blob.core.windows.net` will be used to feth the default Azure credentals with which your host system is utilizing.
```

Comment:
```suggestion
The `storage_account_url` which looks like `https://<myStorageAccountName>.blob.core.windows.net` and it will be used to fetch the default Azure credentials in the host system.
```